docs/documentation/securing_apps/topics/oidc/java/java-adapters.adoc

```diff
@@ -1,13 +1,46 @@
-=== Java adapters
+=== {project_name} Java adapters
+
+[WARNING]
+====
+{project_name} Java Adapters are now in End-Of-Life (EOL) status. They are no longer supported nor receiving security updates.
+It will be removed in future releases.
```

Comment:
This statement is probably too strong. We haven't EOL them quite yet, but rather deprecated. We'll do security updates on adapters until they are fully EOL, but won't do any further development.

Perhaps something like:
```suggestion
{project_name} Java Adapters are now deprecated and will be removed in a future release. No further enhancements or features will be added to the adapter until then.
```